examples/recommender_system/main.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import gym
+import recommender_system_env
+from reinforced_lib import RLib
+from reinforced_lib.agents import EGreedy
+from reinforced_lib.logs import PlotsLogger
+from reinforced_lib.logs.base_logger import SourceType
+
+gym.logger.set_level(40)
+
+
+SEED = 47
+N_EPISODES = 300
+
+
+# Create users preferences
+preferences = {
+    "astronomy": 0.71,      # 0
+    "nature": 0.6,          # 1
+    "cooking": 0.6,         # 2
+    "games": 0.2,           # 3
+    "music": 0.92,          # 4 TOP
+    "sports": 0.4,          # 5
+    "technology": 0.67      # 6
+}
+
+# Create and reset the environment which will simulate users behavior
+env = gym.make("RecommenderSystemEnv-v1", preferences=preferences)
+_ = env.reset(seed=SEED)
+
+# Wrap everything under RLib object with designated agent
+agent_params = {"n_arms": len(preferences), "e": 0.25}
+rl = RLib(
+    agent_type=EGreedy,
+    agent_params=agent_params,
+    no_ext_mode=True,
+    # loggers_type=PlotsLogger,
+    # loggers_sources=[('action', SourceType.METRIC)]
+)
+rl.init(SEED)
+
+# Create data structures to store the history
+actions_trace = np.empty(N_EPISODES)
+rewards_trace = np.empty(N_EPISODES)
```

Comment:
Rozumiem, że te zmienne `*trace` są dlatego, że plotowanie z biblioteki nie działało? Na pewno `actions` dla się w miarę bez problemu plotować, natomiast wykresy dla `reward` to jest jeszcze takie moje TODO. Planowałem to kiedyś dodać, ale zapomniało mi się później, w sumie to byłaby dobra metryka do plotowania. Mógłbym to zaimplementować tak, że jeśli w obserwacjach jest reward to plotuje od razu, a jeśli nie, to sprawdza czy rozszerzenie ma metodę `reward` i wtedy wywołuje tamto.

Reply:
Tak, teraz jak już doprowadziłem plottera do działania, to zostawiłem jedynie historię nagrody. Może jeżeli nam w przyszłości się uda tą funkcjonalność do loggera, to będzie można się tego też pozbyć.